timemachine/fe/single_topology.py

```diff
@@ -853,28 +844,6 @@ def __init__(self, mol_a, mol_b, core, forcefield, lambda_angles=0.4, lambda_tor
         self.src_system = self._setup_end_state_src()
         self.dst_system = self._setup_end_state_dst()
 
-        # setup default parameter interpolation functions
-        self.default_interpolate_harmonic_bond_params_fxn = partial(
-            interpolate_harmonic_bond_params,
-            k_min=0.1,  # ~ BOLTZ * (300 K) / (5 nm)^2
-            lambda_min=0.0,
-            lambda_max=lambda_angles,
-        )
-        self.default_interpolate_harmonic_angle_params_fxn = partial(
-            interpolate_harmonic_angle_params,
-            k_min=0.05,  # ~ BOLTZ * (300 K) / (2 * pi)^2
-            lambda_min=lambda_angles,
-            lambda_max=lambda_torsions,
-        )
-        self.default_interpolate_periodic_torsion_params_fxn = partial(
-            interpolate_periodic_torsion_params,
-            lambda_min=lambda_torsions,
-            lambda_max=1.0,
-        )
-        self.default_interpolate_nonbonded_params_fxn = interpolate.linear_interpolation
-        self.default_interpolate_chiral_atom_params_fxn = interpolate.linear_interpolation
-        self.default_interpolate_chiral_bond_params_fxn = interpolate.linear_interpolation
```

Comment:
Removed this default/override mechanism for now as it wasn't being used and adds a fair amount of clutter. Interpolation functions are for the moment defined inline in `setup_intermediate_state` and `_parameterize_host_guest_nonbonded`.